nodejs/week2/warmup/index.js

```diff
@@ -0,0 +1,24 @@
+const express = require("express");
+const app = express();
+
+app.get("/" , (req, res) => {
+    res.send(`you are home!`);
+});
+
+app.get("/numbers/add/:id" , (req, res) => {
+    res.send(`you entere ${req.params.id}`);
+});
+
+//GET /numbers/multiply/<first number here>/<second number here>. in response send multiplication (first * second).
+app.use("/numbers/multiply" , (req, res) => {
+    const url = req.url;
+    const name = url.split("/");
+    res.send(name[name.length-2] + ' * ' + name[name.length-1]);
```

Comment:
Here we are getting '5 * 5' as a result (if we write 5 in the url). It's good. 
However if you do want to multiply them, and send the actual multiplication, then we would need to convert these strings into number again, and then we are able to multiply them. Here is an example below: 

> app.use("/numbers/multiply" , (req, res) => {
    const url = req.url;
    const name = url.split("/");
    const result = Number(name[name.length-2]) * Number(name[name.length-1]);
    console.log(result);
    res.send(`result is: ${result}`);
});